projects/opendr_ws/src/perception/scripts/rgbd_hand_gesture_recognition.py

```diff
@@ -30,12 +31,14 @@
 
 class RgbdHandGestureNode:
 
-    def __init__(self, input_image_topic="/usb_cam/image_raw", input_depth_image_topic="/usb_cam/image_raw",
+    def __init__(self, input_rgb_image_topic="/kinect2/qhd/image_color_rect",
+                 input_depth_image_topic="/kinect2/qhd/image_depth_rect",
                  gesture_annotations_topic="/opendr/gestures", device="cuda"):
```

Comment:
As a suggestion `gesture_annotations_topic` could be renamed to `gestures_topic` to be more similar to other nodes that for example use `detections_topic` for equivalent arguments.